pkg/aws/iam.go

```diff
@@ -0,0 +1,353 @@
+package aws
```

Comment:
I like they way you structured all this helpers under this file. Maybe we can make the helpers methods of the client? That way we dont need to be passing the clients around. WDYT?

Reply:
I think it is done this way so a mock client can be passed to the functions, but I don't think kubedog really needs to care about that since it's just a collection of godog steps.